person.rb

```diff
@@ -10,8 +14,9 @@ def initialize(age, name = 'Unknown', parrent_permission: true)
     @parrent_permission = parrent_permission
   end
 
-  attr_accessor :name, :age
-  attr_reader :id
+  def add_rental(book, date)
+    Rental.new(book, date, self)
+  end
```

Comment:
- [ ] in order to make this class work you need to add the attr_acessor for Rental and add it in the `initialize` method as an empty array.